otter/auth.py

```diff
@@ -61,6 +61,21 @@
 from otter.util.retry import repeating_interval, retry, retry_times
 
 
+class _DoNothingLogger(BoundLog):
```

Comment:
I don't think that's a safe thing to do.  Production code should not depend on test code.